bluesky/callbacks/best_effort.py

```diff
@@ -518,10 +519,42 @@ def plot_annotations(self):
             if self.ax in self.__labeled:
                 label = '_no_legend_'
             else:
-                label = attr
+                label = '{}={:.2f}'.format(attr, val)
```

Comment:
:+1:

Reply:
It might be nice to make this format string configurable via `__init__` and some attribute, like `label_format='{attr}={val:.2f}'`. The main use case would be to let people increase precision. Might as well give them control over the whole string.

Reply:
I like the idea, will move it to the attribute. I also agree about the precision, we should not stick to 2 decimal points.